openlibrary/plugins/worksearch/code.py

```diff
@@ -477,7 +477,7 @@ def GET(self):
 
         self.redirect_if_needed(i)
 
-        if 'isbn' in i and all(not v for k, v in i.items() if k != 'isbn'):
+        if 'isbn' in i:
```

Comment:
This fixes search (if isbn is present, redirects gracefully, even if multiple other query params present)